Assets/Example/Scripts/CountlyEntryPoint.cs

```diff
@@ -393,3 +408,15 @@ public void OnNotificationClicked(string message, int index)
         Debug.Log("[Countly Example] OnNoticicationClicked: " + message + ", index: " + index);
     }
 }
+
+public class CustomDataType
```

Comment:
No need to create a custom data type for this. Just use any already available one

Reply:
then i am going to use counlty object.